app/src/main/java/space/taran/arkretouch/presentation/drawing/EditManager.kt

```diff
@@ -291,16 +291,9 @@ class EditManager {
         drawPaint.value.color = color
         _currentPaintColor.value = color
         if (oldColors.isNotEmpty() && oldColors.last() != color) {
-            val size = oldColors.size
-            when {
-                size < 20 -> {
-                    _oldColors.add(color)
-                }
-                size == 20 -> {
-                    _oldColors.removeFirst()
-                    _oldColors.add(color)
-                }
-            }
+            _oldColors.add(color)
+            if (oldColors.size > MAX_COLOR_SIZE)
+                _oldColors.removeFirst()
```

Comment:
I can imagine really rare corner case when `_oldColors = _oldColors.takeLast(MAX_COLORS_AMOUNT)` will be better, but ok ;-)

Reply:
Yes, what is it?